tap_harvest/__init__.py

```diff
@@ -460,15 +460,19 @@ def do_sync():
 
     LOGGER.info("Sync complete")
 
+def do_discover():
+    print('{"streams":[]}')
 
 def main_impl():
     args = utils.parse_args(REQUIRED_CONFIG_KEYS)
     CONFIG.update(args.config)
     global AUTH  # pylint: disable=global-statement
     AUTH = Auth(CONFIG['client_id'], CONFIG['client_secret'], CONFIG['refresh_token'])
     STATE.update(args.state)
-    do_sync()
-
+    if args.discover:
+        do_discover()
+    else:
+        do_sync()
```

Comment:
I think we found this change was unnecessary.  I would prefer not to change the tap if we don't need to.

Reply:
I think this change is necessary. I think it is needed for the tap to run discovery prior to the test running.